src/ol_concourse/pipelines/open_edx/mfe/pipeline.py

```diff
@@ -82,6 +83,7 @@ def mfe_params(
         "CONTACT_URL": open_edx.contact_url,
         "CSRF_TOKEN_API_PATH": "/csrf/api/v1/token",
         "DISPLAY_FEEDBACK_WIDGET": open_edx.display_feedback_widget,
+        "ENABLE_CERTIFICATE_PAGE": open_edx.enable_certificate_page,
```

Comment:
This will result in `"False"` or `"True"` being passed as the parameter. As long as that is acceptable to the MFE build that will work, but if the intent is to have lowercased values then this would need to be stringified and lowercased.
```suggestion
        "ENABLE_CERTIFICATE_PAGE": str(open_edx.enable_certificate_page).lower(),
```

Reply:
I am not sure, it's a boolean value for `ENABLE_CERTIFICATE_PAGE` which has been set in mfe [here](https://github.com/openedx/frontend-app-course-authoring/pull/1223/files#diff-e9cbb0224c4a3d23a6019ba557e0cd568c1ad5e1582ff1e335fb7d99b7a1055dR37)

> This will result in "False" or "True" being passed as the parameter. 
Why would it be converted to string? I have set it to boolean type..